server/routes/index.ts

```diff
@@ -7,7 +7,7 @@ import removePatientRoutes from './removePatient'
 import homepageRoutes from './homepage'
 
 import { Services } from '../services'
-import addPrisonerRoutes from './addPrisoner'
+import addPrisonerRoutes from './addPatient'
```

Comment:
add patient makes more sense here since we are adding them to restricted patients.  Will rename the routes at a later stage, don't want to make PR too big...